apps/meteor/client/views/marketplace/AppDetailsPage/tabs/AppDetails/AppDetails.tsx

```diff
@@ -18,6 +21,8 @@ type AppDetailsProps = {
 	};
 };
 
+const GET_ADDONS_LINK = 'https://go.rocket.chat/i/get-addons';
```

Comment:
Move this link to https://github.com/RocketChat/Rocket.Chat/blob/develop/apps/meteor/client/views/admin/subscription/utils/links.ts so it can be utilised in other places too

Reply:
ok